### PR TITLE
Feat/5967  improve space settings and dashboard

### PR DESCRIPTION
**Pull request by gbonaspetti**
#5967 

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)
- [x] Make sure that:
  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified
- [x] Original authors of the features included in a multi-feature branch (maintenance fixes -> develop, security fixes -> develop, …) should be part of the reviewers, especially if you encountered merge conflicts.

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [x] Manual, quality tests have been done

### CODE REVIEW

**Review by grignards — CHANGES_REQUESTED** [review]

File: frontend_app_workspace_advanced/src/component/SpaceDescription.jsx [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+              <textarea
+                className='workspace_advanced__description__text__textarea'
+                hidden
+                id={props.textareaId}
+                onChange={props.onChangeDescription}
+                placeholder={props.t("Space's description")}
+                value={props.description}
+              />
+            </div>
+            {props.isAutoCompleteActivated && props.autoCompleteItemList.length > 0 && (
```
[the remark this answers is not in the document]
Comment (gbonaspetti) [reply frontend_app_workspace_advanced/src/component/SpaceDescription.jsx:56]: Fixed

File: frontend_app_workspace_advanced/src/component/SpaceDescription.jsx [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+  useEffect(() => {
+    if (showEditionMode) {
+      globalThis.wysiwyg(
+        `#${props.textareaId}`,
+        props.lang,
+        props.onChangeDescription,
+        props.onTinyMceInput,
+        props.onTinyMceKeyDown,
+        props.onTinyMceKeyUp,
+        props.onTinyMceSelectionChange
```
[the remark this answers is not in the document]
Comment (gbonaspetti) [reply frontend_app_workspace_advanced/src/component/SpaceDescription.jsx:22]: Fixed

File: frontend_app_workspace_advanced/src/component/SpaceDescription.jsx [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+      )
+    }
+  }, [props.lang, showEditionMode])
```
Comment (Skylsmoi) [inline frontend_app_workspace_advanced/src/component/SpaceDescription.jsx:25]: if `lang` change, shouldn't `tinymceRemove` be called before calling `globalThis.wysiwyg`?

Comment (gbonaspetti) [reply frontend_app_workspace_advanced/src/component/SpaceDescription.jsx]: Fixed

**Review by Skylsmoi — APPROVED** [review]

**Review by Millefeuille42 — APPROVED** [review]

**Comment by Haskaris** [comment]
After testing, we've decided that it would be better to move the `Space settings` button to the bottom of the button list
image
to
image

Can you do the change?
Or should we take the lead here?

**Comment by Haskaris** [comment]
Perfect! 
Thank you for the contribution! 😄

**Pull request merged**